src/pages/guides/one-key-flow.mdx

```diff
@@ -25,6 +25,8 @@ import Tabs from "@theme/Tabs";
 
 ## Introduction
 
+---
+
 Adding Web3Auth Plug and Play SDKs to your application is a great way to authenticate users and reconstruct their private key, and to reconstruct the
```

Comment:
Starting looks odd to me. Can we align this with our other guides?

Reply:
This specific description was added to tell users the flow which was currently and how we are changing it using the one key flow. The context was added after a discussion and thus this is different as compared to other guides